shared/chat/conversation/info-panel/notifications/index.js

```diff
@@ -133,6 +109,35 @@ export const Notifications = (props: Props) => (
     <Box style={styleSaveState}>
       <SaveStateComponent saveState={props.saveState} />
```

Comment:
I think this should remain visible when muting, right?

Reply:
Oops, yes! Done.